app/Http/routes.php

```diff
@@ -34,6 +33,15 @@
 
 Route::get('mentions', 'TweetsController@index');
 
+Route::get('users','UsersController@index');
+Route::get('users/create','UsersController@create');
+Route::post('users','UsersController@store');
+Route::get('users/{id}','UsersController@show');
+Route::delete('users/{id}','UsersController@destroy');
+Route::get('users/{id}/edit','UsersController@edit');
+Route::PATCH('users/{id}','UsersController@update');
+Route::DELETE('users','UsersController@delete');
+
```

Comment:
use Route::resource
a 1 liner and generates all those for you
you can check the generated routes with php artisan route:list and adapt to the ones generated